src/ContainerCheckpointModal.jsx

```diff
@@ -1,89 +1,71 @@
-                                       ignoreRootFS: this.state.ignoreRootFS
-                                   });
-                               }}>
-                           {_("Checkpoint")}
-                       </Button>
-                       <Button variant="link" isDisabled={this.state.inProgress}
-                               onClick={Dialogs.close}>
-                           {_("Cancel")}
-                       </Button>
-                   </>}
-            >
-                <Form isHorizontal>
-                    <Checkbox label={_("Keep all temporary checkpoint files")} id="checkpoint-dialog-keep"
-                                  name="keep" isChecked={this.state.keep} onChange={this.handleChange} />
-                    <Checkbox label={_("Leave running after writing checkpoint to disk")}
-                                  id="checkpoint-dialog-leaveRunning" name="leaveRunning"
-                                  isChecked={this.state.leaveRunning} onChange={this.handleChange} />
-                    <Checkbox label={_("Support preserving established TCP connections")}
-                                  id="checkpoint-dialog-tcpEstablished" name="tcpEstablished"
-                                  isChecked={this.state.tcpEstablished} onChange={this.handleChange} />
-                    <Checkbox label={_("Do not include root file-system changes when exporting")}
-                                  id="checkpoint-dialog-ignoreRootFS" name="ignoreRootFS"
-                                  isChecked={this.state.ignoreRootFS} onChange={this.handleChange} />
-                </Form>
-            </Modal>
-        );
-    }
-}
+    return (
+        <Modal isOpen
+               showClose={false}
+               position="top" variant="medium"
+               title={cockpit.format(_("Checkpoint container $0"), containerWillCheckpoint.Names)}
+               footer={<>
+                   <Button variant="primary" isDisabled={inProgress}
+                           isLoading={inProgress}
+                           onClick={handleCheckpointContainer}>
+                       {_("Checkpoint")}
+                   </Button>
+                   <Button variant="link" isDisabled={inProgress}
+                           onClick={Dialogs.close}>
+                       {_("Cancel")}
+                   </Button>
+               </>}
+        >
+            <Form isHorizontal>
+                <Checkbox label={_("Keep all temporary checkpoint files")} id="checkpoint-dialog-keep"
+                              name="keep" isChecked={keep} onChange={(checked, _) => setKeep(checked)} />
+                <Checkbox label={_("Leave running after writing checkpoint to disk")}
+                              id="checkpoint-dialog-leaveRunning" name="leaveRunning"
+                              isChecked={leaveRunning} onChange={(checked, _) => setLeaveRunning(checked)} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1146-20221123-171443-906cf86c-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)